frontend/nc-games/src/components/reviewList.jsx

```diff
@@ -1,29 +1,51 @@
 import React, {useState, useEffect} from "react";
```

Comment:
The name of this file and navigation.jsx themselves need to be changed too, please! (in the file navigator)